PSMDB-929 Release notes for 4.2.17-17

nastena1606 opened the pull request:
modified:   source/conf.py
	modified:   source/data_at_rest_encryption.rst
	new file:   source/release_notes/4.2.17-17.rst
	modified:   source/release_notes/index.rst

nastena1606 committed:
PSMDB-929 Release notes for 4.2.17-17

modified:   source/conf.py
	modified:   source/data_at_rest_encryption.rst
	new file:   source/release_notes/4.2.17-17.rst
	modified:   source/release_notes/index.rst

spron-in commented on source/release_notes/4.2.17-17.rst:
```diff
@@ -0,0 +1,41 @@
+It supports MongoDB 4.2.16 and 4.2.17 protocols and drivers.
+
+
+MongoDB Release Highlights
+=============================
+
+* The ``libsasl2`` dependencies are added for DEB-based operating systems to enable GSSAPI and Kerberos authentication to work out of the box after installing MongoDB
+* The number of modify updates in WiredTiger is now limited to prevent performance issues
+* Errors related to proxy connections receive new error codes which don't change the behavior of clients' drivers.
+
+Find the full list of changes in the  `MongoDB 4.2.16 Community Edition <https://docs.mongodb.com/manual/release-notes/4.2/#4.2.16---sep-13--2021>`_ and MongoDB 4.2.17 Community Edition <>`_ release notes.
+
+Percona Server for MongoDB Release Highlights
+=============================================
+
+* The ``mongod`` user is not deleted during the re-installation of Percona Server for MongoDB on RPM-based operating systems. This preserves the permissions to the MongoDB data directory for the ``mongod`` user as its user ID and group ID remain unchanged.
+* RPM packages now include the ``cyrus-sasl`` modules to enable LDAP authentication via native LDAP against Active Directory
```
I suggest to keep only one Release Highlights section. Our goal is to deliver open source software to community and customers and if MongoDB corp team did most of the job - so be it :)

I feel that our improvement and a bug fix are not that significant for highlights. So in here it should be okay to keep only the ones which came from MongoDB corp, and our tickets should be okay to be under corresponding sections. 

I really like the value oriented description that you created in Release Highlights, we might reuse it in Improvements and Bugs Fixed sections. What do you think?

nastena1606 replied:
Fixed: removed MongoDB highlights section, making it the only one. 
Extended PSMDB Jira issue subjects

kwilktx commented:
Made a few additions to the 4.2.x highlights section in the doc on the g-Drive - here:
https://docs.google.com/document/d/1Sp0_jLGmf26uNvBSMtgIQmN9RFXfi4PPiwFCRDJlLW8/edit#

igorsol commented on .netlify/state.json:
```diff
@@ -0,0 +1,3 @@
+{
+	"siteId": "af1034bc-c9f2-4d8a-b0a3-a7cca4264af5"
+}
```
PLease add newline at the end of file

nastena1606 replied:
This file is auto-generated. Updated the .gitignore to not track it

nastena1606 committed:
PSMDB-929 Release notes for 4.2.17-17

modified:   source/conf.py
	modified:   source/data_at_rest_encryption.rst
	new file:   source/release_notes/4.2.17-17.rst
	modified:   source/release_notes/index.rst

nastena1606 committed:
Merge branch 'nastena1606-PSMDB-929-Release-notes-4.2.17-17' into PSMDB-929-Release-notes-4.2.17-17
